packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -0,0 +1,31 @@
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const getBaseButtonStyles = ({
+  color,
+  hoverColor,
+  activeColor,
+}: Omit<StyledBaseButtonProps, 'children' | 'onClick'>): string => `
+  display:flex;
+  flex-direction: row;
+  height: 48px;
+  cursor: pointer;
+  background-color: ${color};
+  border: 2px solid ${color};
+  color: white;
+  border-radius: 2px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  align-self: center;
+  padding: 0px 20px 0px 20px;
+  &:hover {
```

Comment:
shall we try out using https://styled-components.com/docs/advanced#style-objects. Think it's better than plan interpolated strings. Realised late after releasing Typography.

Reply:
Yup, lets do this. Much better